pyproject.toml

```diff
@@ -97,6 +97,14 @@ extended_tasks = [
   "openai", # llm as a judge using openai models
 ]
 s3 = ["s3fs"]
+multilingual = [
+    "stanza",
+    "pyvi",
+    "pythainlp",
+    "jieba",
+    "indic-nlp-library",
+    "kiwipiepy",
```

Comment:
Do we really need all these added deps? We try to maintain lighteval light, so if it's only for a function or 2 we rather copy the code and mention its origin in comments.